modules/images-configuration-image-registry-settings-hcp.adoc

```diff
@@ -0,0 +1,133 @@
+// Module included in the following assemblies:
+//
+// * openshift_images/image-configuration-hcp.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="images-configuring-image-registry-settings-hcp_{context}"]
+= Configuring image registry settings for {hcp-title}
+
+Verify that your image registry is configured in the cluster details by running the following command:
+
+[NOTE]
+====
+You can change any registry related parameter which will trigger a rollout across all machinepools (all machinepool nodes will be recreated, following pod draining from each node).
+====
+
+.Procedure
+
+. Create {hcp-title} clusters with image registry:
+
++
+[source,terminal]
+----
+$ rosa create cluster —cluster-name=<cluster_name> --sts --mode=auto \
+   --hosted-cp --operator-roles-prefix <operator_role_prefix> \
+   --oidc-config-id <id_of_oidc_configuration> \
+   --subnet-ids=<public_subnet_id>,<private_subnet_id> \
+   --registry-config-insecure-registries <insecure_registries> \
+   --registry-config-allowed-registries <allowed_registries> \
+   --registry-config-allowed-registries-for-import <registry_name:insecure> \
+   --registry-config-additional-trusted-ca <additional_trusted_ca_file>
+----
++
+[NOTE]
+====
+When using the `allowedRegistries`, `blockedRegistries`, or `insecureRegistries` parameter, you can specify an individual repository within a registry. For example: `reg1.io/myrepo/myapp:latest`.
+
+Insecure external registries should be avoided to reduce possible security risks.
+Parameters `allowedRegistries`, `blockedRegistries` are mutually exclusive.
+====
+
+.Verification
+
+*  Run the `rosa describe` command to verify that your image registry is enabled in the cluster details by running the following command:
```

Comment:
You've actually got 2 total steps in this `.Verification` procedure, so this should be a `.` (also not the extra space, tho that probably wasn't messing anything up:
```suggestion
. Run the `rosa describe` command to verify that your image registry is enabled in the cluster details by running the following command:
```
When this was a `*`, it was making the next step below (which is a `.`) start the numbering over at `1`.